src/requestLogger.js

```diff
@@ -17,6 +18,10 @@ class RequestLogger {
     }
   }
 
+  initialize() {
```

Comment:
Does the naming make sense here? It already has a constructor, why another initialize method?

I think it would be easier for clients to understand if we just explain that it sets invocationId which can be used for tracking purposes.

Reply:
I figured it wouldn't be scalable to call it `setInvocationId`, `setInvocation` is that better?

Reply:
I guess if the value is provided by `RequestLogger` itself then `startInvocation`.

Reply:
`startInvocation` it is.